apps/studio/components/interfaces/Database/Replication/DestinationRow.tsx

```diff
@@ -128,7 +128,6 @@ const DestinationRow = ({
 
     try {
       await stopPipeline({ projectRef, pipelineId: pipeline.id })
-      await deletePipeline({ projectRef, pipelineId: pipeline.id })
```

Comment:
just to double check if this was intentional? if yes then can probably remove the use of `useDeletePipelineMutation`

Reply:
This is intentional. Added a comment in the code here, explaining why (cascade delete)